### PR TITLE
Enable drowpdown tab in the patient menu

### PR DESCRIPTION
**Pull request by matrix-amiel**
Hi.

Here I added option to make dropdown menu in the patient file (In the new style the amount of the tabs is limited in the case you have a lot of modules.. like us ;) ).
From now if there are children in the patient menu json, they will created in dropdown, like the main menu (but with regular bootstrap style and only for single level) 

image

Thanks
Amiel

### CODE REVIEW

File: library/menu/src/PatientMenuRole.php [hunk 1]
```diff
@@ -178,19 +180,22 @@ public function displayHorizNavBarMenu()
...
+                    $list .= '<a class="oe-bold-black"  href="' . attr($link) . '" onclick="' . $children_value->on_click .'"> ' . text($children_value->label) . ' </a>';
                     $list .= '</li>';
                 }
-                echo $list. "\r\n";
+                $list .= '</ul>';
             } else {
                 $link = ($value->pid != "true") ? $value->url : $value->url . attr($pid);
                 $class = isset($value->class) ? $value->class : '';
-                $list = '<li class="oe-bold-black ' . attr($class) . '" id="' . attr($value->menu_id) . '">';
+                $list = '<li class="oe-bold-black' . attr($class) . '" id="' . attr($value->menu_id) . '">';
```
Comment (bradymiller) [inline library/menu/src/PatientMenuRole.php]: don't you need this space between class names?

**Comment by bradymiller** [comment]
hi @matrix-amiel , Noted 1 potential issue with spacing between class names. If this is addressed and this is testing well for you, then will bring it into the codebase. thanks!, -brady

**Comment by matrix-amiel** [comment]
Fixed..
Tested and working well.
Thank you!

**Comment by bradymiller** [comment]
hi @matrix-amiel , Bringing this into the codebase. thanks for the improvement! -brady

**Pull request merged**